modules/oe_theme_content_event/tests/src/FunctionalJavascript/LivestreamDisplayDisclosingTest.php

```diff
@@ -0,0 +1,140 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_theme_content_event\FunctionalJavascript;
+
+use Drupal\Core\Datetime\DrupalDateTime;
+use Drupal\datetime\Plugin\Field\FieldType\DateTimeItemInterface;
+use Drupal\filter\Entity\FilterFormat;
+use Drupal\FunctionalJavascriptTests\WebDriverTestBase;
+
+/**
+ * Test livestream description and link disclosing.
+ *
+ * @group batch3
+ *
+ * @group oe_theme_content_event
+ */
+class LivestreamDisplayDisclosingTest extends WebDriverTestBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  protected static $modules = [
+    'config',
+    'system',
+    'path',
+    'block',
+    'datetime_testing',
+    'oe_theme_helper',
+    'oe_theme_content_event',
+  ];
+
+  /**
+   * {@inheritdoc}
+   */
+  protected $defaultTheme = 'stark';
+
+  /**
+   * {@inheritdoc}
+   */
+  public function setUp(): void {
+    parent::setUp();
+    // Enable and set OpenEuropa Theme as default.
+    $this->container->get('theme_installer')->install(['oe_theme']);
+    $this->config('system.theme')->set('default', 'oe_theme')->save();
+    $this->container->set('theme.registry', NULL);
+
+    // Rebuild the ui_pattern definitions to collect the ones provided by
+    // oe_theme itself.
+    \Drupal::service('plugin.manager.ui_patterns')->clearCachedDefinitions();
+
+    FilterFormat::create([
+      'format' => 'full_html',
+      'name' => 'Full HTML',
+    ])->save();
+  }
+
+  /**
+   * Tests that Event livestreaming link will be visible exactly on time.
+   */
+  public function testContentWithoutInPageNav(): void {
+    $static_time = new DrupalDateTime('now', DateTimeItemInterface::STORAGE_TIMEZONE);
+    $start_date = (clone $static_time)->modify('+10 days');
+    // Create an Event node with required fields only.
+    /** @var \Drupal\node\Entity\Node $node */
+    $node = $this->container->get('entity_type.manager')
+      ->getStorage('node')
+      ->create([
+        'type' => 'oe_event',
+        'title' => 'Test event node',
+        'oe_event_type' => 'http://publications.europa.eu/resource/authority/public-event-type/COMPETITION_AWARD_CEREMONY',
+        'oe_teaser' => 'Event teaser',
+        'oe_subject' => 'http://data.europa.eu/uxp/1000',
+        'oe_event_status' => 'as_planned',
+        'oe_event_dates' => [
+          'value' => $start_date->format(DateTimeItemInterface::DATETIME_STORAGE_FORMAT),
+          'end_value' => $start_date->format(DateTimeItemInterface::DATETIME_STORAGE_FORMAT),
+        ],
+        'oe_event_online_dates' => [
+          'value' => $start_date->format(DateTimeItemInterface::DATETIME_STORAGE_FORMAT),
+          'end_value' => $start_date->format(DateTimeItemInterface::DATETIME_STORAGE_FORMAT),
```

Comment:
Why are we setting the end dates to the same as the start date?

Reply:
It doesn't have a big value so end_date had a random value.